<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolsPanelClass</class>
 <widget class="QMainWindow" name="ToolsPanelClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>947</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ToolsPanel</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QScrollArea" name="glWidgetArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>921</width>
      <height>551</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>919</width>
       <height>549</height>
      </rect>
     </property>
     <widget class="QToolButton" name="pr1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>25</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QToolButton" name="pr2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>25</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QToolButton" name="pr3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>25</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QToolButton" name="pr4">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>25</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
     <widget class="QToolButton" name="pr5">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>40</y>
        <width>25</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QToolButton" name="pr6">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>40</y>
        <width>25</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
     <widget class="QToolButton" name="zoomM">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>150</y>
        <width>25</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="opgl.qrc">
        <normaloff>:/icons/zoom_out.png</normaloff>:/icons/zoom_out.png</iconset>
      </property>
     </widget>
     <widget class="QToolButton" name="zoomP">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>130</y>
        <width>25</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="opgl.qrc">
        <normaloff>:/icons/zoom_in.png</normaloff>:/icons/zoom_in.png</iconset>
      </property>
     </widget>
     <widget class="QToolButton" name="snap">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>270</y>
        <width>25</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="opgl.qrc">
        <normaloff>:/icons/camera.png</normaloff>:/icons/camera.png</iconset>
      </property>
     </widget>
     <widget class="QToolButton" name="openFileTool">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>210</y>
        <width>25</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="opgl.qrc">
        <normaloff>:/icons/file.png</normaloff>:/icons/file.png</iconset>
      </property>
     </widget>
     <widget class="QToolButton" name="openDefaultTool">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>160</y>
        <width>25</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="opgl.qrc">
        <normaloff>:/icons/doDefMethod.png</normaloff>:/icons/doDefMethod.png</iconset>
      </property>
     </widget>
     <widget class="QToolButton" name="xLeft">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>120</y>
        <width>25</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="opgl.qrc">
        <normaloff>:/icons/arrow_left.png</normaloff>:/icons/arrow_left.png</iconset>
      </property>
     </widget>
     <widget class="QToolButton" name="xRight">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>110</y>
        <width>25</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="opgl.qrc">
        <normaloff>:/icons/arrow_right.png</normaloff>:/icons/arrow_right.png</iconset>
      </property>
     </widget>
     <widget class="QToolButton" name="yLeft">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>160</y>
        <width>25</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="opgl.qrc">
        <normaloff>:/icons/arrow_left.png</normaloff>:/icons/arrow_left.png</iconset>
      </property>
     </widget>
     <widget class="QToolButton" name="yRight">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>160</y>
        <width>25</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="opgl.qrc">
        <normaloff>:/icons/arrow_right.png</normaloff>:/icons/arrow_right.png</iconset>
      </property>
     </widget>
     <widget class="QToolButton" name="zLeft">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>190</y>
        <width>25</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="opgl.qrc">
        <normaloff>:/icons/arrow_left.png</normaloff>:/icons/arrow_left.png</iconset>
      </property>
     </widget>
     <widget class="QToolButton" name="zRight">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>190</y>
        <width>25</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="opgl.qrc">
        <normaloff>:/icons/arrow_right.png</normaloff>:/icons/arrow_right.png</iconset>
      </property>
     </widget>
     <widget class="QCheckBox" name="axis">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>300</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="opgl.qrc">
        <normaloff>:/icons/3d-axis.png</normaloff>:/icons/3d-axis.png</iconset>
      </property>
     </widget>
     <widget class="QCheckBox" name="grid">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>260</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="opgl.qrc">
        <normaloff>:/icons/grid.png</normaloff>:/icons/grid.png</iconset>
      </property>
     </widget>
     <widget class="QCheckBox" name="figure">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>280</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="opgl.qrc">
        <normaloff>:/icons/figure.png</normaloff>:/icons/figure.png</iconset>
      </property>
     </widget>
     <widget class="QToolButton" name="exit">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>50</y>
        <width>25</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="opgl.qrc">
        <normaloff>:/icons/exit.png</normaloff>:/icons/exit.png</iconset>
      </property>
     </widget>
     <widget class="QToolButton" name="cancel">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>60</y>
        <width>25</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="opgl.qrc">
        <normaloff>:/icons/cancel.png</normaloff>:/icons/cancel.png</iconset>
      </property>
     </widget>
     <widget class="QToolButton" name="author">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>340</y>
        <width>25</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="opgl.qrc">
        <normaloff>:/icons/about_author.png</normaloff>:/icons/about_author.png</iconset>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QScrollArea" name="glLineArea">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>560</y>
      <width>541</width>
      <height>31</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>539</width>
       <height>29</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="blueValue">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="greenValue">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>590</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="redValue">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>590</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>947</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="openAction">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="openDefaultAction">
   <property name="text">
    <string>Открыть по умолчанию</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="opgl.qrc"/>
 </resources>
 <connections/>
</ui>
